env/production/replica_count.yaml

```diff
@@ -71,5 +71,5 @@ metadata:
   name: document-download-api-hpa
   namespace: notification-canada-ca
 spec:
-  minReplicas: 1
+  minReplicas: 2
```

Comment:
Is this a requirement of the new upgrade? Or we just want to make this more reliable?

Reply:
When the upgrade happens, K8s will try and move over only one pod at a time. If there's only one pod running, it will terminate first and then move over, which will introduce downtime. With two running, when it terminates the first one, the other will handle the traffic until the new one comes up.

Reply:
ah yes, makes total sense, thanks!

Reply:
Although thinking about it, Kubernetes usually spins up a new pod before killing the old one. But I guess this is different for upgrading its own pod?